SQL/Archive/2018-02-28_issues_add_cascade.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE issues_history DROP FOREIGN KEY `fk_issues_comments_1`;
+ALTER TABLE issues_history ADD CONSTRAINT `fk_issues_comments_1` FOREIGN KEY (`issueID`) REFERENCES `issues` (`issueID`) ON DELETE CASCADE ON UPDATE RESTRICT;
+
+ALTER TABLE issues_comments DROP FOREIGN KEY `fk_issue_comments_1`;
+ALTER TABLE issues_comments ADD CONSTRAINT `fk_issue_comments_1` FOREIGN KEY (`issueID`) REFERENCES `issues` (`issueID`) ON DELETE CASCADE ON UPDATE RESTRICT;
+
+ALTER TABLE issues_comments_history DROP FOREIGN KEY `fk_issues_comments_history`;
+ALTER TABLE issues_comments_history ADD CONSTRAINT `fk_issues_comments_history` FOREIGN KEY (`issueCommentID`) REFERENCES `issues_comments` (`issueCommentID`) ON DELETE CASCADE ON UPDATE RESTRICT;
```

Comment:
The name of the first 2 constraint are the same except for 1 plural 's'.  Should not the name of the first constraint be 'fk_issues_history_1'?

Even if table name should always be singular, constraint2 should also have an 's' to follow table name.
Maybe you could follow the new modeling standard for naming FK constraint.
Same for 00-00-00 script.

Reply:
its not really the purpose of this PR. i only want to add the on delete and on update clauses so didn't think to change the name of the constraint that was already there. i could easily change it but then there are also other things we could change to follow the new modelling standard

Reply:
I agree, making the changes in this PR will trigger many more changes to remain consistent with the issue_tracker architecture... might be too involved for the changes that are done here